packages/suite/src/views/wallet/coinmarket/exchange/components/ExchangeForm/Inputs/FiatInput/index.tsx

```diff
@@ -25,25 +26,30 @@ const FiatInput = () => {
         trigger,
         updateSendCryptoValue,
         setValue,
+        getValues,
     } = useCoinmarketExchangeFormContext();
 
-    const error = errors.outputs && errors.outputs[0] ? errors.outputs[0].fiat : undefined;
+    const amountError = errors.outputs && errors.outputs[0] ? errors.outputs[0].amount : undefined;
+    const fiatError = errors.outputs && errors.outputs[0] ? errors.outputs[0].fiat : undefined;
```

Comment:
just an option how to write it more compact, definitely matter of taste so use it if you like it
```suggestion
    const amountError = errors.outputs?.[0]?.amount;
    const fiatError = errors.outputs?.[0]?.fiat;
```

or 
```suggestion
   const outputsErrors = errors.outputs?.[0] ?? {};
   const { amount: amountError, fiat: fiatError } = outputsErrors;
```

Reply:
Agreed. Done.